rdkit/Chem/UnitTestRegistrationHash.py

```diff
@@ -0,0 +1,641 @@
+"""
```

Comment:
I think it would be good to add:
- unit tests which test thjngs we don't expect to work (ie where exceptions might be thrown) like overly large tautomer values
-  tests for molecules with isotopes, including large isotope values

Reply:
I'm not sure what you mean by "overly large tautomer values"

Are you thinking about situations in which the tautomerhash doesn't capture tautomerization?

Reply:
Hmm, TBH I'm not sure what I meant by that... will edit the comment.